datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -174,6 +180,10 @@ func (r *integrationGcpStsResource) Create(ctx context.Context, request resource
 	state.DelegateAccountEmail = types.StringValue(delegateEmail)
 
 	attributes, diags := r.buildIntegrationGcpStsRequestBody(ctx, &state)
+	if !state.ClientEmail.IsNull() {
+		attributes.SetClientEmail(state.ClientEmail.ValueString())
+	}
```

Comment:
Just FYI: the create endpoint will fail without `client_email` (it's required), so maybe we don't even send it?

Reply:
Spoke offline. We prob could make an optimization here in bailing on the request since this will fail. But this is what existed, so good with it for now.

Some follow-ups on the dogweb side:
- `client_email` is required, but can be blank, and ICS will reject that request, which means dogweb returns 500. Not good, should be 400
- dogweb should probably do the validation for `client_email` so we don't even go to ICS